guides/common/modules/proc_using-project-as-a-source-of-content-for-imagebuilder.adoc

```diff
@@ -0,0 +1,21 @@
+[id="using-project-as-a-source-of-content-for-imagebuilder"]
+= Using {Project} as a source of content for {LoraxCompose}
```

Comment:
The title is very long. I wonder whether it's possible to shorten it. 

Also, the content reads like an RH Image Builder procedure. Only step 1 is related to {Project} -- recording the repo URL. Maybe this procedure should be documented in the {LoraxCompose} docs and linked from Satellite.

Reply:
Perhaps `{Project} as a content source for {LoraxCompose}`?

Reply:
@apinnick I agree it should be in the LoraxCompose docs. I'll discuss it with Eliane, the RHEL TW.